releasenotes/notes/add-inverse-ecr-e03720252a0c9c1e.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add the method :class:`.ECRGate.inverse` that returns the same ECRGate,
+    since it is self inverse.
```

Comment:
```suggestion
    Add the method :meth:`.ECRGate.inverse` that returns the same :class:`.ECRGate`,
    since it is self inverse.
```

Reply:
We can fix this in the final pass over the release notes for merge, but this text is a bit imprecise and kind of implies that the same class _instance_ is returned, rather than a new instance of the same class.  That has implications for mutability.  It's super minor, though.